src/lib/modules/ens/index.js

```diff
@@ -281,8 +281,10 @@ class ENS {
   }
 
   registerSubDomain(defaultAccount, subDomainName, reverseNode, address, secureSend, cb) {
-    ENSFunctions.registerSubDomain(this.ensContract, this.registrarContract, this.resolverContract, defaultAccount,
-      subDomainName, this.registration.rootDomain, reverseNode, address, this.logger, secureSend, cb);
+    this.events.request("blockchain:get", (web3) => {
+      ENSFunctions.registerSubDomain(web3, this.ensContract, this.registrarContract, this.resolverContract, defaultAccount,
```

Comment:
Register subdomain now requires web3 to be passed to it instead of getting the global one. 
(Death to global web3 💀 )